src/components/demos/write/Demo.js

```diff
@@ -55,91 +92,105 @@ export default function WriteDemo(props) {
   const nav = useNavigate()
   const state = React.useContext(Context)
   const { update, writeDemo: demoState } = state
-  const { theme } = demoState
+  const { theme, showHelp, usernamePasswordPairs, output, showResponse } =
```

Comment:
why are we moving everything onto the context? This doesn't feel like global state

Reply:
wanted to keep component cleaner and we also call `update` context method after each api response so why not to use something that is already there? We do not care much about performance, it's outstanding and I don't think that for this app it will become an issue, knowing the size of global state. Ideally would like to break into demo specific context or 3rd party